packages/api/src/schemas/envConfig.json

```diff
@@ -33,7 +33,41 @@
         },
         "values": {
           "type": "object",
-          "description": "Configs to customize the Teams app manifest."
+          "description": "Configs to customize the Teams app manifest.",
+          "properties": {
+            "appName": {
```

Comment:
Shall we mark the properties like `appName` as required? Otherwise, the mustache render would be failed for `{{config.manifest.values.appName.short}}`?

Reply:
We can mark the `appName` as required, but we need to generate a default value for `appName` in `config.{env}.json` when we create a new environment. And user have the flexibility to customize this property.